python-client/giskard/scanner/llm/minority_bias_detector.py

```diff
@@ -0,0 +1,144 @@
+from dataclasses import dataclass
+from typing import List, Sequence
+from langchain import PromptTemplate
+
+import pandas as pd
+from detoxify import Detoxify
```

Comment:
We cannot import this at global level, it would break if not installed.

Reply:
DONE